service-seed-without-router/play-service/test/controllers/health/HealthControllerTest.java

```diff
@@ -40,13 +39,13 @@ public void testGetHealthSuccess() {
         Map<String, Object> reqMap = new HashMap<>();
         reqMap.put("accept", "yes");
         Result result = testHelper.performTest("/health", "GET", reqMap, headerMap);
-        assertTrue(testHelper.getResponseStatus(result) == Response.Status.OK.getStatusCode());
+        assertTrue(testHelper.getResponseStatus(result) == 200);
```

Comment:
any relevance to these changes? If not, please revert.

Reply:
As I removed this dependency 
 <dependency>	
            <groupId>org.sunbird</groupId>	
            <artifactId>mw-service</artifactId>	
            <version>1.0-SNAPSHOT</version>	
   </dependency>
it was throwing an error missing  this dependency
<dependency>
            <groupId>javax.ws.rs</groupId>
            <artifactId>javax.ws.rs-api</artifactId>
            <version>2.1.1</version>
        </dependency>

Now added this in pom.xml